python/nano/src/bigdl/nano/deps/onnxruntime/tensorflow/tensorflow_onnxruntime_model.py

```diff
@@ -78,7 +86,10 @@ def __call__(self, *args, **kwargs):
                 kwargs[name] = value
         for param in self._call_fn_args_backup[len(inputs):len(self._forward_args)]:
             inputs.append(kwargs[param])
-        return self.call(*inputs)
+        outputs = self.call(*inputs)
```

Comment:
If I understand right, here returns original onnx output(a list) ? so maybe it contains _nesting_level at first (not start from 0) ? 
It's just a bit confusing whether this way of adding lists works in all cases

Reply:
No, here `outputs` is a single Tensor, not onnx output.

Reply:
`tf.py_function(func, ..., Tout=tf.float32)` will convert the output of func to a single Tensor